dev/clint/src/clint/rules.py

```diff
@@ -230,3 +230,50 @@ def check(node: ast.Call) -> bool:
                     and first_elem.value == "mlflow"
                 )
         return False
+
+
+def _is_abstract_method(node: ast.FunctionDef | ast.AsyncFunctionDef) -> bool:
+    return any(
+        (isinstance(d, ast.Name) and d.id == "abstractmethod")
+        or (
+            isinstance(d, ast.Attribute)
+            and isinstance(d.value, ast.Name)
+            and d.value.id == "abc"
+            and d.attr == "abstractmethod"
+        )
+        for d in node.decorator_list
+    )
+
+
+class InvalidAbstractMethod(Rule):
+    def _id(self) -> str:
+        return "MLF0015"
+
+    def _message(self) -> str:
+        return (
+            "Abstract method should only contain a single statement/expression, "
+            "and it must be `pass`, `...`, or a docstring."
+        )
+
+    @staticmethod
+    def check(node: ast.FunctionDef | ast.AsyncFunctionDef) -> bool:
+        return _is_abstract_method(node) and (
+            # Does this abstract method have multiple statements/expressions?
+            len(node.body) > 1
+            # This abstract method has a single statement/expression.
+            # Check if it's `pass`, `...`, or a docstring. If not, it's invalid.
+            or not (
+                # pass
+                isinstance(node.body[0], ast.Pass)
+                or (
+                    isinstance(node.body[0], ast.Expr)
+                    and isinstance(node.body[0].value, ast.Constant)
+                    and (
+                        # `...`
+                        node.body[0].value.value is ...
+                        # docstring
+                        or isinstance(node.body[0].value.value, str)
+                    )
+                )
+            )
+        )
```

Comment:
/ai suggest 5 more names for this rule.

Reply:
/ai I'm looking for an adjective I can replace `invalid` with. `invalid` sounds ambiguous.